### PR TITLE
Create the new upgrade category manual test plan and add initial test plan

### PR DESCRIPTION
**Pull request by TachunLin**
Create the new upgrade category manual test plan and add major test
cases including:

1. Upgrade from new network design (1.1.0 and after)
2. Upgrade from traditional network design (before v1.1.0)
3. Upgrade in fully airpagged network
4. Upgrade with ipv6 DHCP network
5. Rejoin node machine after upgrade

### CODE REVIEW

**Review by noahgildersleeve — CHANGES_REQUESTED** [review]
@TachunLin You need to add an `_index.md` file in the upgrade tests directory for it to properly separate in Hugo. Otherwise you end up with it just going into the overall list.
Greenshot 2022-11-14 09 41 17

You can look at something basic like we have in the UI directory for a template.

```
---
title: UI
---
These are some basic tests for Harvester UI (the web app itself).
```

File: docs/content/manual/upgrade/upgrade-from-new-network-design.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+1. Open Networks page
+1. Create vlan `1` network on `mgmt` cluster network
+1. Create another vlan network on `vlan` cluster network in step 3
+    ![image](https://user-images.githubusercontent.com/29251855/201615070-969e6c34-1a8e-4540-a750-32f7f38b585c.png)
+1. Create images with different OS distribution
+1. Create several virtual machines, set network to `management-network` or available `vlan` 
+1. Create virtual machine on different target node
+1. Setup NFS or S3 backup target in settings
+1. Back each virtual machines
+1. Shutdow all virtual machines
```
Comment (noahgildersleeve) [inline docs/content/manual/upgrade/upgrade-from-new-network-design.md:48]: Should be `Shutdown`

File: docs/content/manual/upgrade/upgrade-from-new-network-design.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+1. Offline upgrade to traget version, refer to https://docs.harvesterhci.io/v1.1/upgrade/automatic
```
Comment (noahgildersleeve) [inline docs/content/manual/upgrade/upgrade-from-new-network-design.md:49]: Should be `target version`

File: docs/content/manual/upgrade/upgrade-from-new-network-design.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+1. Import Harvester into the Rancher cluster
+1. Adding node after the upgrade
```
Comment (noahgildersleeve) [inline docs/content/manual/upgrade/upgrade-from-new-network-design.md:51]: Should be `Add node`

File: docs/content/manual/upgrade/upgrade-from-traditional-network-design.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+1. Create images with different OS distribution
+1. Create several virtual machines, set network to `management-network` or available `vlan` 
+1. Create virtual machine on different target node
+1. Setup NFS or S3 backup target in settings
+1. Back each virtual machines
+1. Shutdow all virtual machines
+1. Offline upgrade to traget version, refer to https://docs.harvesterhci.io/v1.1/upgrade/automatic
+1. Import Harvester into the Rancher cluster
+1. Adding node after the upgrade
```
Comment (noahgildersleeve) [inline docs/content/manual/upgrade/upgrade-from-traditional-network-design.md:50]: Should be `Add node`

File: docs/content/manual/upgrade/upgrade-from-new-network-design.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+
+## Expected Results
+1. Can completely upgrade Harvester to specific version
+1. All pods is running correctly
```
Comment (noahgildersleeve) [inline docs/content/manual/upgrade/upgrade-from-new-network-design.md:56]: Should be `All pods are running`

File: docs/content/manual/upgrade/upgrade-from-new-network-design.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+1. Check can display Monitoring Chart
```
Comment (noahgildersleeve) [inline docs/content/manual/upgrade/upgrade-from-new-network-design.md:57]: Maybe rephrase this to just `Can display monitoring chart`, or `Check that you can display`

File: docs/content/manual/upgrade/upgrade-from-traditional-network-design.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
+---
+title: Upgrade Harvester from traditonal cluster network design (before v1.1.0)
+---
+
+
+## Category: 
+* Upgrade Harvester
+
+## Environment requirement
+1. Network environment have available VLAN id setup on DHCP server can be assigned to Harvester
```
Comment (noahgildersleeve) [inline docs/content/manual/upgrade/upgrade-from-traditional-network-design.md:10]: See previous comment about this line

File: docs/content/manual/upgrade/upgrade-from-traditional-network-design.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+1. Network have at least two NICs
```
Comment (noahgildersleeve) [inline docs/content/manual/upgrade/upgrade-from-traditional-network-design.md:11]: Should be `has at least`

File: docs/content/manual/upgrade/upgrade-from-traditional-network-design.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+1. Suggest each Harvester node have at lease 6 core CPU, 16GB memory and 200GB disk
```
Comment (noahgildersleeve) [inline docs/content/manual/upgrade/upgrade-from-traditional-network-design.md:12]: Should be `node has at least`

File: docs/content/manual/upgrade/upgrade-from-traditional-network-design.md [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+1. For other VLAN id testing, enable network on settings, select correct network interface e.g `enps0f1`
+    ![image](https://user-images.githubusercontent.com/29251855/201609582-d9e129f2-1c1e-416f-b878-9df4903ad2e2.png)
+1. Create a `vlan91` network with available id `91` on Networks page (id number depends on your available network setting)
+    ![image](https://user-images.githubusercontent.com/29251855/201609880-22bf619c-d215-403a-8299-63c62934cfe2.png)
+
+1. Create images with different OS distribution
+1. Create several virtual machines, set network to `management-network` or available `vlan` 
+1. Create virtual machine on different target node
+1. Setup NFS or S3 backup target in settings
+1. Back each virtual machines
```
Comment (noahgildersleeve) [inline docs/content/manual/upgrade/upgrade-from-traditional-network-design.md:46]: Should be `backup`

File: docs/content/manual/upgrade/upgrade-from-traditional-network-design.md [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+1. Shutdow all virtual machines
```
Comment (noahgildersleeve) [inline docs/content/manual/upgrade/upgrade-from-traditional-network-design.md:47]: Should be `Shutdown`

File: docs/content/manual/upgrade/upgrade-from-traditional-network-design.md [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+1. Offline upgrade to traget version, refer to https://docs.harvesterhci.io/v1.1/upgrade/automatic
```
Comment (noahgildersleeve) [inline docs/content/manual/upgrade/upgrade-from-traditional-network-design.md:48]: Should be `Target`

File: docs/content/manual/upgrade/upgrade-from-traditional-network-design.md [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+1. Import Harvester into the Rancher cluster
+1. Adding node after the upgrade
+
+
+## Expected Results
+1. Can completely upgrade Harvester to specific version
+1. All pods is running correctly
+1. Check can display Monitoring Chart 
+- Prometheus dashboard
+- VM metrics
```
Comment (noahgildersleeve) [inline docs/content/manual/upgrade/upgrade-from-traditional-network-design.md:58]: Should be indented to be sub-list

File: docs/content/manual/upgrade/fully-airgapped-upgrade.md [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+---
+
+## Category: 
+* Upgrade Harvester
+
+## Environment requirement
+1. Airgapped Network without internet connectivity
+1. Network environment have available VLAN id setup on DHCP server can be assigned to Harvester
+1. Network have at least two NICs
+1. Suggest each Harvester node have at lease 6 core CPU, 16GB memory and 200GB disk
```
Comment (khushboo-rancher) [inline docs/content/manual/upgrade/fully-airgapped-upgrade.md:12]: Why 6 core CPU, 16GB memory, when we have mentioned as 8-core processor minimum for testing and 32 GB minimum in our official docs?

File: docs/content/manual/upgrade/fully-airgapped-upgrade.md [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+
+## Expected Results
+1. Can completely upgrade Harvester to specific version
+1. All pods is running correctly
+1. Check can display Monitoring Chart 
+   - Prometheus dashboard
+   - VM metrics
+1. Can access dashboard by VIP
+1. Can use original password to login
+1. Can start VM in running status
```
Comment (khushboo-rancher) [inline docs/content/manual/upgrade/fully-airgapped-upgrade.md:61]: Can create new VMs also.

File: docs/content/manual/upgrade/ipv6-dhcp-upgrade.md [hunk 16]
```diff
@@ -0,0 +1,79 @@
...
+1. Select DHCP node ip and DHCP vip during the installation 
+1. Create another VMs and use the ipv6 network. (workload)
+1. ISO install v1.0.3 in join mode
+1. Select DHCP node ip and DHCP vip during the installation
+1. Offline upgrade to master release, refer to https://docs.harvesterhci.io/v1.1/upgrade/automatic
+1. Check host IP on Host page 
+1. Check the ip allocated to node `kubectl get nodes -o json | jq '.items[].status.addresses'` 
+
+
+## Expected Results
```
Comment (khushboo-rancher) [inline docs/content/manual/upgrade/ipv6-dhcp-upgrade.md:57]: Can you include the P0 checks as you have mentioned in the fully-airgapped-upgrade test case?

File: docs/content/manual/upgrade/rejoin-node-after-upgrade.md [hunk 17]
```diff
@@ -0,0 +1,46 @@
...
+
+## Expected Results
+1. Verify after Harvester upgrade complete, we **can rejoin** the management node and agent node back correctly. 
+
+1. Can re-join the management node after upgrade 
+  ![image](https://user-images.githubusercontent.com/29251855/196159969-9c8acb11-b9fe-4501-94d5-74545579ef4d.png)
+
+1. re-join the agent node after upgrade 
+  ![image](https://user-images.githubusercontent.com/29251855/196164183-20c4498a-c4c6-4033-a527-d3c8ff84f8aa.png)
+
```
Comment (khushboo-rancher) [inline docs/content/manual/upgrade/rejoin-node-after-upgrade.md:45]: Can create new vms on the newly joined node.
Can restore vms/migrate on the newly joined node. etc

File: docs/content/manual/upgrade/upgrade-from-new-network-design.md [hunk 18]
```diff
@@ -0,0 +1,73 @@
...
+---
+
+
+## Category: 
+* Upgrade Harvester
+
+## Environment requirement
+1. Network environment have available VLAN id setup on DHCP server can be assigned to Harvester
+1. Network have at least two NICs
+1. Suggest each Harvester node have at lease 6 core CPU, 16GB memory and 200GB disk
```
Comment (khushboo-rancher) [inline docs/content/manual/upgrade/upgrade-from-new-network-design.md:12]: Same as above comment, have the minimum required hardware.

File: docs/content/manual/upgrade/fully-airgapped-upgrade.md [hunk 19]
```diff
@@ -0,0 +1,70 @@
...
+1. Create virtual machine on different target node
+1. Setup NFS or S3 backup target in settings
+1. Back each virtual machines
+1. Shutdow all virtual machines
+1. Offline upgrade to traget version, refer to https://docs.harvesterhci.io/v1.1/upgrade/automatic
+
+
+## Expected Results
+1. Can completely upgrade Harvester to specific version
+1. All pods is running correctly
```
[the remark this answers is not in the document]
Comment (TachunLin) [reply docs/content/manual/upgrade/fully-airgapped-upgrade.md]: Appreciated for the checking. Update to the correct wording

File: docs/content/manual/upgrade/fully-airgapped-upgrade.md [hunk 20]
```diff
@@ -0,0 +1,70 @@
...
+1. For other VLAN id testing, enable network on settings, select correct network interface e.g `enps0f1`
+    ![image](https://user-images.githubusercontent.com/29251855/201609582-d9e129f2-1c1e-416f-b878-9df4903ad2e2.png)
+1. Create a `vlan91` network with available id `91` on Networks page (id number depends on your available network setting)
+    ![image](https://user-images.githubusercontent.com/29251855/201609880-22bf619c-d215-403a-8299-63c62934cfe2.png)
+
+1. Create images with different OS distribution
+1. Create several virtual machines, set network to `management-network` or available `vlan` 
+1. Create virtual machine on different target node
+1. Setup NFS or S3 backup target in settings
+1. Back each virtual machines
```
[the remark this answers is not in the document]
Comment (TachunLin) [reply docs/content/manual/upgrade/fully-airgapped-upgrade.md]: Appreciated for the checking. Update this typo.

File: docs/content/manual/upgrade/fully-airgapped-upgrade.md [hunk 21]
```diff
@@ -0,0 +1,70 @@
...
+1. Shutdow all virtual machines
```
[the remark this answers is not in the document]
Comment (TachunLin) [reply docs/content/manual/upgrade/fully-airgapped-upgrade.md]: Appreciated for the checking. Update this typo.

File: docs/content/manual/upgrade/ipv6-dhcp-upgrade.md [hunk 22]
```diff
@@ -0,0 +1,79 @@
...
+    <ip family="ipv6" address="fd7d:844d:3e17:f3ae::1" prefix="64">
+        <dhcp>
+        <range start="fd7d:844d:3e17:f3ae::100" end="fd7d:844d:3e17:f3ae::1ff"/>
+        </dhcp>
+    </ip>
+    </network>
+    ```
+    
+1. Change the bridge name to a new one 
+    ![image](https://user-images.githubusercontent.com/29251855/201565649-ba54d6f7-3540-4a4b-ad66-ee4a77cfbff1.png)
```
[the remark this answers is not in the document]
Comment (TachunLin) [reply docs/content/manual/upgrade/ipv6-dhcp-upgrade.md:43]: Appreciated for the checking. Add a new line between line 41 and 42 to separate image

File: docs/content/manual/upgrade/upgrade-from-traditional-network-design.md (hunk 9 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-from-traditional-network-design.md]: Appreciated for the checking. Update this typo.

File: docs/content/manual/upgrade/upgrade-from-traditional-network-design.md (hunk 10 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-from-traditional-network-design.md]: Appreciated for the checking. Update this typo.

File: docs/content/manual/upgrade/upgrade-from-traditional-network-design.md (hunk 11 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-from-traditional-network-design.md]: Appreciated for the checking. Update this typo.

File: docs/content/manual/upgrade/upgrade-from-traditional-network-design.md (hunk 12 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-from-traditional-network-design.md]: Appreciated for the checking. Update this typo.

File: docs/content/manual/upgrade/upgrade-from-traditional-network-design.md (hunk 13 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-from-traditional-network-design.md]: Appreciated for the checking. Indent the following check items
   - Prometheus dashboard
   - VM metrics

File: docs/content/manual/upgrade/fully-airgapped-upgrade.md (hunk 14 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/fully-airgapped-upgrade.md]: I checked our official document, we did not mention the CPU, disk and memory requirement for upgrade test. 
It's more of from the past testing experience to prevent unexpected failure. 
I would remove this line of requirement in all related upgrade test plan

File: docs/content/manual/upgrade/fully-airgapped-upgrade.md (hunk 15 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/fully-airgapped-upgrade.md:60]: Add this check point to the bottom

File: docs/content/manual/upgrade/ipv6-dhcp-upgrade.md (hunk 16 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/ipv6-dhcp-upgrade.md:58]: Append all P0 checks in the fully-airgapped-upgrade test case to the ipv6-dhcp-upgrade test case

File: docs/content/manual/upgrade/rejoin-node-after-upgrade.md (hunk 17 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/rejoin-node-after-upgrade.md]: Add these check points to the bottom to expected result

File: docs/content/manual/upgrade/upgrade-from-new-network-design.md (hunk 18 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-from-new-network-design.md]: Remove the line of minimum required hardware

File: docs/content/manual/upgrade/upgrade-from-new-network-design.md [hunk 23]
```diff
@@ -0,0 +1,73 @@
...
+
+## Expected Results
+1. Can completely upgrade Harvester to specific version
+1. All pods is running correctly
+1. Check can display Monitoring Chart 
+   - Prometheus dashboard
+   - VM metrics
+1. Can access dashboard by VIP
+1. Can use original password to login
+1. Can start VM in running status
```
[the remark this answers is not in the document]
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-from-new-network-design.md:61]: Add this check point to the bottom

File: docs/content/manual/upgrade/upgrade-from-traditional-network-design.md [hunk 24]
```diff
@@ -0,0 +1,72 @@
...
+1. Can access dashboard by VIP
+1. Can use original password to login
+1. Can start VM in running status
```
[the remark this answers is not in the document]
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-from-traditional-network-design.md:60]: Add this check point to the bottom

**Comment by TachunLin** [comment]
Thanks for team's check, I have updated according to review request. 
And add two more test plan according to the v1.1.1 release testing plan


1. Upgrade from v1.0.3 with bonded NICs on network
2. Upgrade with slow HDD disks

**Review by noahgildersleeve — CHANGES_REQUESTED** [review]
There are still some typos and things that need to be changed. The Github interface for the changes gets kind of buggy if there's a lot. You're going to have to expand it several times to see all of them. It's kind of unintuitive.

File: docs/content/manual/upgrade/upgrade-with-HDD-disk.md [hunk 25]
```diff
@@ -0,0 +1,61 @@
+---
+title: Upgrade Harvester with HDD Disks
+---
+
+
+## Category: 
+* Upgrade Harvester
+
+## Environment requirement
+1. Network environment have available VLAN id setup on DHCP server can be assigned to Harvester
```
Comment (noahgildersleeve) [inline docs/content/manual/upgrade/upgrade-with-HDD-disk.md:10]: Should be `network environment has`

File: docs/content/manual/upgrade/upgrade-with-HDD-disk.md [hunk 26]
```diff
@@ -0,0 +1,61 @@
...
+1. Network have at least two NICs
```
Comment (noahgildersleeve) [inline docs/content/manual/upgrade/upgrade-with-HDD-disk.md:11]: Should be `Network has`

**Review by noahgildersleeve — CHANGES_REQUESTED** [review]
Some small changes on final upgrade scenario. Otherwise looks good.

File: docs/content/manual/upgrade/fully-airgapped-upgrade.md (hunk 14 above)
Comment (khushboo-rancher) [reply docs/content/manual/upgrade/fully-airgapped-upgrade.md]: I think to have a Harvester cluster up and running, we need to meet the min requirements https://docs.harvesterhci.io/v1.1#hardware-requirements, so its okay to include the minimum requirement here as well.

File: docs/content/manual/upgrade/ipv6-dhcp-upgrade.md [hunk 27]
```diff
@@ -0,0 +1,79 @@
...
+
+
+## Verification Steps
+1. Create a VM and use the ipv6 network. (workload)
+1. ISO install v1.0.3 in create mode
```
[the remark this answers is not in the document]
Comment (TachunLin) [reply docs/content/manual/upgrade/ipv6-dhcp-upgrade.md]: Thanks for the check. I correct to the following statement for clearer instruction

1. Create a VM and use the customized ipv6 virtual network. (workload)
1. Doing an Harvester v1.0.3 ISO install with `create` new mode
1. Select DHCP node ip and DHCP vip during the installation 
1. Create another VMs use the customized ipv6 virtual network
1. Doing an Harvester v1.0.3 ISO install with `join` mode

File: docs/content/manual/upgrade/ipv6-dhcp-upgrade.md [hunk 28]
```diff
@@ -0,0 +1,79 @@
...
+      "address": "192.168.101.162",
+      "type": "InternalIP"
+    },
+    {
+      "address": "harv103-master",
+      "type": "Hostname"
+    }
+  ]
+  
+  ```
```
[the remark this answers is not in the document]
Comment (TachunLin) [reply docs/content/manual/upgrade/ipv6-dhcp-upgrade.md]: Thanks for the check, I have increased the indent to better display the numbering order

File: docs/content/manual/upgrade/rejoin-node-after-upgrade.md [hunk 29]
```diff
@@ -0,0 +1,46 @@
+---
+title: Rejoin node machine after Harveter upgrade
```
[the remark this answers is not in the document]
Comment (TachunLin) [reply docs/content/manual/upgrade/rejoin-node-after-upgrade.md]: Thanks for the check, correct the typo to "Harvester upgrade"

File: docs/content/manual/upgrade/rejoin-node-after-upgrade.md [hunk 30]
```diff
@@ -0,0 +1,46 @@
...
+* Related issues: [#2655](https://github.com/harvester/harvester/issues/2655) 
+[BUG] reinstall 1st node
+
+
+## Category: 
+* Upgrade Harvester
+
+
+## Environment requirement
+1. Network environment have available VLAN id setup on DHCP server can be assigned to Harvester
```
[the remark this answers is not in the document]
Comment (TachunLin) [reply docs/content/manual/upgrade/rejoin-node-after-upgrade.md]: Thanks for the check, I modified the line 14 of Environment requirement to better understanding 

1. Network environment has available VLAN id setup on DHCP server
1. DHCP server has setup the IP range can allocate to above VLAN id
1. Harvester node can route to DHCP server through VLAN id to retrieve IP address

File: docs/content/manual/upgrade/rejoin-node-after-upgrade.md [hunk 31]
```diff
@@ -0,0 +1,46 @@
...
+1. Network have at least two NICs
```
[the remark this answers is not in the document]
Comment (TachunLin) [reply docs/content/manual/upgrade/rejoin-node-after-upgrade.md]: Thanks for the check, correct the typo

File: docs/content/manual/upgrade/upgrade-from-new-network-design.md [hunk 32]
```diff
@@ -0,0 +1,73 @@
+---
+title: Upgrade Harvester from new cluster network design (after v1.1.0)
+---
+
+
+## Category: 
+* Upgrade Harvester
+
+## Environment requirement
+1. Network environment have available VLAN id setup on DHCP server can be assigned to Harvester
```
[the remark this answers is not in the document]
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-from-new-network-design.md]: Thanks for the check, I modified the line 10 of Environment requirement to better understanding 

1. Network environment has available VLAN id setup on DHCP server
1. DHCP server has setup the IP range can allocate to above VLAN id
1. Harvester node can route to DHCP server through VLAN id to retrieve IP address

File: docs/content/manual/upgrade/upgrade-from-new-network-design.md [hunk 33]
```diff
@@ -0,0 +1,73 @@
...
+1. Network have at least two NICs
```
[the remark this answers is not in the document]
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-from-new-network-design.md]: Thanks for the check, correct the typo

File: docs/content/manual/upgrade/upgrade-from-new-network-design.md (hunk 1 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-from-new-network-design.md]: Thanks for the check, correct the typo.

File: docs/content/manual/upgrade/upgrade-from-new-network-design.md (hunk 3 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-from-new-network-design.md]: Thanks for the check, correct the typo.

File: docs/content/manual/upgrade/upgrade-from-new-network-design.md (hunk 5 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-from-new-network-design.md]: Thanks for the check, correct the typo.

File: docs/content/manual/upgrade/upgrade-from-new-network-design.md (hunk 6 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-from-new-network-design.md]: Thanks for the check, rephrase this line to `Check that can display the Monitoring Chart`.

File: docs/content/manual/upgrade/upgrade-from-traditional-network-design.md (hunk 7 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-from-traditional-network-design.md]: Thanks for the check, I modified the line 10 of Environment requirement to better understanding 

1. Network environment has available VLAN id setup on DHCP server
1. DHCP server has setup the IP range can allocate to above VLAN id
1. Harvester node can route to DHCP server through VLAN id to retrieve IP address

File: docs/content/manual/upgrade/upgrade-from-traditional-network-design.md (hunk 8 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-from-traditional-network-design.md]: Thanks for the check, correct the typo.

File: docs/content/manual/upgrade/upgrade-from-traditional-network-design.md (hunk 4 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-from-traditional-network-design.md]: Thanks for the check, correct the typo.

File: docs/content/manual/upgrade/upgrade-with-HDD-disk.md (hunk 25 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-with-HDD-disk.md]: Thanks for the check, correct the typo

File: docs/content/manual/upgrade/upgrade-with-HDD-disk.md (hunk 26 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-with-HDD-disk.md]: Thanks for the check, correct the typo

**Comment by TachunLin** [comment]
Really appreciated to @noahgildersleeve and @khushboo-rancher for the detailed review. 
I have updated the typos and some phrases to make clearer instructions. 
Please let me know whether I miss anything. Thank you.

File: docs/content/manual/upgrade/ipv6-dhcp-upgrade.md (hunk 27 above)
[the remark this answers is not in the document]
Comment (noahgildersleeve) [reply docs/content/manual/upgrade/ipv6-dhcp-upgrade.md]: You might consider changing `Doing an Harvester v1.0.3 ISO` to something like `Install Harvester from v1.0.3 ISO`. It's a little clearer. If not then you should change `an Harvester` to `a Harvester`

File: docs/content/manual/upgrade/upgrade-from-traditional-network-design.md (hunk 10 above)
Comment (noahgildersleeve) [reply docs/content/manual/upgrade/upgrade-from-traditional-network-design.md]: This does not seem to be updated

**Review by noahgildersleeve — CHANGES_REQUESTED** [review]
Just a small typo. Other than that it looks good.

**Review by khushboo-rancher — APPROVED** [review]
LGTM

File: docs/content/manual/upgrade/upgrade-from-new-network-design.md [hunk 34]
```diff
@@ -0,0 +1,75 @@
...
+1. Select NICs on the uplink page 
+1. Open Networks page
+1. Create vlan `1` network on `mgmt` cluster network
+1. Create another vlan network on `vlan` cluster network in step 3
+    ![image](https://user-images.githubusercontent.com/29251855/201615070-969e6c34-1a8e-4540-a750-32f7f38b585c.png)
+1. Create images with different OS distribution
+1. Create several virtual machines, set network to `management-network` or available `vlan` 
+1. Create virtual machine on different target node
+1. Setup NFS or S3 backup target in settings
+1. Back each virtual machines
```
Comment (noahgildersleeve) [inline docs/content/manual/upgrade/upgrade-from-new-network-design.md:48]: This should be backup

File: docs/content/manual/upgrade/upgrade-from-traditional-network-design.md (hunk 10 above)
Comment (noahgildersleeve) [reply docs/content/manual/upgrade/upgrade-from-traditional-network-design.md]: fixed.

**Review by noahgildersleeve — CHANGES_REQUESTED** [review]
Just the one typo and other than that it LGTM

File: docs/content/manual/upgrade/upgrade-from-new-network-design.md (hunk 34 above)
Comment (TachunLin) [reply docs/content/manual/upgrade/upgrade-from-new-network-design.md]: Thanks for the check. Correct the typo to `Backup each virtual machines`

**Review by noahgildersleeve — APPROVED** [review]
LGTM
